.github/workflows/npm-publish.yml

```diff
@@ -0,0 +1,21 @@
+# This workflow will publish a package to npmjs.com when a release is created
+# For more information see: https://docs.github.com/en/actions/publishing-packages/publishing-nodejs-packages
+
+name: Node.js Package
+
+on: push
```

Comment:
Why we don't use the:
```
on:
  release:
    types: [published]
 ```
suggested by the mentioned https://docs.github.com/en/actions/publishing-packages/publishing-nodejs-packages?

Reply:
I guess that in this case, you would have to create the release manually. This script will do it automatically.
https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository#creating-a-release

Reply:
Ok, thanks 👍